src/box/lua/config/utils/schema.lua

```diff
@@ -1872,15 +1874,27 @@ local function set_schema_annotations(schema, annotations)
             schema.description = description or schema.description
     end
 
+    -- Set the validator for the given schema node based on the field
+    -- path accumulated in ctx.path.
+    local function set_validator(schema, ctx)
+        local field_path = table.concat(ctx.path, '.')
+        local validate = validators[field_path]
+        -- The schema object may already have a validate function set by
+        -- `enterprise_edition` or provided by `schema.set`. To preserve
+        -- the existing validate function, use `chain2` to combine the
+        -- new validator with the one that is already set.
+        schema.validate = funcutils.chain2(validate, schema.validate)
```

Comment:
I think that the order of the chaining should different: the existing function first, then the external one. See more detailed thoughts in https://github.com/tarantool/tarantool/pull/10694#discussion_r1871693053.

Reply:
This code has been removed, so it is no longer relevant.